styleguide/_static/issue-templates/bug.md

```diff
@@ -0,0 +1,76 @@
+---
+name: 🐞 Bug Report 
+about: File a bug report 
+title: 🐞 <Insert Title> 
+labels: bug
+assignees: ''
+
+---
+
+### Checklist
+In order to improve the time, how long a bugfix does take, please make sure you address as much as possible of the following checklist.
+
+- [ ] I have checked the issue tracker for the same issue and I haven't found one similar.
+- [ ] I have reproduced the issue with at least the latest released version of \<PROJECT-NAME\>.
+
+**Tip: Adding screenshots can help(s) to improve clarity.**
+
+# Summary
+<!---
+Describe the bug: A clear and concise description of what the bug is.
+-->
+
+## Reproduce the Issue
+<!---
+Describe how to reproduce the bug, e.g.:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+-->
```

Comment:
I guess, this is an example, and below is where user should put it?

Reply:
yes